### DISCUSSION TITLE
OSError: cannot load library 'gobject-2.0-0': error 0x7e.  Additionally, ctypes.util.find_library() did not manage to locate a library called 'gobject-2.0-0'

### DISCUSSION THREAD

**Discussion by uday-felix** [saleor/saleor#10137] [Q&A]
while installing the project as mentioned in the installation steps, I encountered with the below issue.

OSError: cannot load library 'gobject-2.0-0': error 0x7e.  Additionally, ctypes.util.find_library() did not manage to locate a library called 'gobject-2.0-0'

Tools:
Python : 3.8 or 3.9 
gtk3

Tried all the support from various links, but nothing worked. Can someone please help me out.

**Comment by karolkielecki** [comment]
Hi @uday-felix, have you tried the following [solution](https://www.pythonfixing.com/2021/11/fixed-oserror-cannot-load-library-error.html)?
